### ISSUE TITLE
OpCodeGen.pl script fails on Windows due to syntax errors

### ISSUE THREAD

**Issue by 4creators** [dotnet/coreclr#12018]
Environment:

Windows 10 Pro with several lang packs, active en-US
Command prompt VS 2017 v15.2
fresh clone of coreclr git repo
installed Strawberry perl v5.24.0 in user path 

Reported error:

```
e:\src\ms\coreclr\coreclr.git\src\inc>perl OpCodeGen.pl
Missing braces on \o{} at OpCodeGen.pl line 58, within string
Missing braces on \o{} at OpCodeGen.pl line 87, within string
Missing braces on \o{} at OpCodeGen.pl line 105, within string
Missing braces on \o{} at OpCodeGen.pl line 123, within string
Missing braces on \o{} at OpCodeGen.pl line 141, within string
Execution of OpCodeGen.pl aborted due to compilation errors.
```

**Comment by danmosemsft** [comment]
Thanks @4creators . Would you like to offer a PR to fix?

**Comment by 4creators** [comment]
@danmosemsft Sure, will do it

**Comment by 4creators** [comment]
@danmosemsft  I have made a PR #12040 with some comments. Problem seems to be more complex and deeper than it seems from the surface. Pls look at the PR comments.

**Comment by 4creators** [comment]
Fixed with commit 116a6cc7dd0ff0912251f7d1febed36460ea11cd
